src/movement/control_system.cpp

```diff
@@ -284,19 +300,13 @@ namespace robosub
          * Update the X, Y, and Z positions of the state vector. Note that X
          * and Y position are currently unknown and always set to zero.
          */
-        state_vector[0] = 0; //x
-        state_vector[1] = 0; //y
-        state_vector[2] = depth_msg.depth; //z
+        state_vector[2] = depth_msg.depth;
```

Comment:
I just want to be sure that my suggested change is still valid with the underlying math.